druid-shell/src/backend/shared/xkb/keycodes.rs

```diff
@@ -8,6 +8,7 @@ use super::xkbcommon_sys::*;
 pub fn map_key(keysym: u32) -> Key {
     use Key::*;
     match keysym {
+        XKB_KEY_v => Key::Character("v".to_string()),
```

Comment:
so this fixes paste..... but it implies deeper issues in the keymapping logic for x11/wayland

Reply:
We can have keep this for now